autoscheduler/autoscheduler/config/config.py

```diff
@@ -0,0 +1,23 @@
+import os
+import json
+
+
+def get_prop(prop):
+    """
+    get property from postgres-info.json, or create that file if needed
+    """
+    infoFile = os.path.dirname(__file__) + "/postgres-info.json"
+    try:
+        info = json.load(open(infoFile, "r"))
+        read_prop = info.get(prop)
+        if prop == "user" and read_prop == "":
+            print(f"Please configure {os.path.abspath(infoFile)} to include "
+            "your postgres user and password.")
+            exit()
```

Comment:
Pylint says to use `sys.exit()` rather than `exit()`. See https://stackoverflow.com/questions/19747371/python-exit-commands-why-so-many-and-when-should-each-be-used

Reply:
fixed